pkg/templateservicebroker/openservicebroker/server/route.go

```diff
@@ -48,6 +49,13 @@ func Route(container *restful.Container, path string, b api.Broker) {
 	ws.Route(ws.GET("/service_instances/{instance_id}/last_operation").To(shim(lastOperation)))
 	ws.Route(ws.PUT("/service_instances/{instance_id}/service_bindings/{binding_id}").To(shim(bind)))
 	ws.Route(ws.DELETE("/service_instances/{instance_id}/service_bindings/{binding_id}").To(shim(unbind)))
+
+	routes := ws.Routes()
+	for i, route := range routes {
+		// CHECK: ok to override resource with route.Path?
+		routes[i].Function = metrics.InstrumentRouteFunc(route.Method, route.Path, "", "cluster", route.Function)
```

Comment:
@deads2k I'd like to be able to instrument API calls against the TSB - do you think it's feasible to use "apiserver_request_count" for this?  Problem is that "apiserver_request_count" expects resource/subresource whereas we're dealing with slightly more arbitrary API paths - current worst case is something like "/service_instances/{instance_id}/service_bindings/{binding_id}"

Reply:
It's a little weird, but since its your own servers, I suppose its fine.